src/v2/view-builder/views/EnrollProfileUpdateView.js

```diff
@@ -0,0 +1,52 @@
+import { _, loc, View } from 'okta';
+import { BaseFooter, BaseForm, BaseView } from '../internals';
+import hbs from 'handlebars-inline-precompile';
+import { getSkipSetupLink } from '../utils/LinksUtil';
+
+const Body = BaseForm.extend({
+
+  className: 'profile-update',
+
+  title() {
+    return loc('oie.profile.additional.title', 'login');
+  },
+  
+  save() {
+    return loc('enroll.choices.submit.finish', 'login');
+  },
+
+  getUISchema() {
+    const uiSchemas = BaseForm.prototype.getUISchema.apply(this, arguments);
+    uiSchemas.forEach(input => {
+      if (input.name === 'userProfile.secondEmail') {
+        input.explain = View.extend({
+          template: hbs`{{i18n code="oie.profile.additional.secondemail.subtitle" bundle="login" $1="<strong>$1</strong>"}}`
+        });
+      }
+    });
+    
+    return uiSchemas;
+  },
+
+  initialize() {
+    BaseForm.prototype.initialize.apply(this, arguments);
+  },
+});
+
+const Footer = BaseFooter.extend({
+  className: 'auth-footer side-by-side-links',
+  links: function() {
+    const uiSchema = this.options.currentViewState.uiSchema;
+    const optionalParams = uiSchema.filter(item => item.required === false);
```

Comment:
Since `uiSchema` is always be defined, consider destructuring so it is easier to read:

```suggestion
    const { uiSchema } = this.options.currentViewState;
    const optionalParams = uiSchema.filter(item => item.required === false);
```

Reply:
Unused `const` `optionalParams`

Reply:
used at line 41..

Reply:
😅  Right in front of my eyes..